### PR TITLE
Attached stdout, and stderr to test result.

### PR DESCRIPTION
**Pull request by Haplois**

### CODE REVIEW

File: test/Microsoft.TestPlatform.TestUtilities/IntegrationTestBase.cs [hunk 1]
```diff
@@ -120,6 +123,25 @@ public IntegrationTestBase()
         public void InvokeVsTest(string arguments)
         {
             this.ExecuteVsTestConsole(arguments, out this.standardTestOutput, out this.standardTestError, out this.runnerExitCode);
+
+            if (TestContext != null)
+            {
+                var outputDir = Path.Combine(Path.GetTempPath(), TestContext.TestName, Guid.NewGuid().ToString());
+                if (!Directory.Exists(outputDir))
+                {
+                    Directory.CreateDirectory(outputDir);
```
Comment (nohwnd) [inline test/Microsoft.TestPlatform.TestUtilities/IntegrationTestBase.cs:132]: This call is idempotent, you don't need to check if the directory exists.

File: test/Microsoft.TestPlatform.TestUtilities/IntegrationTestBase.cs [hunk 2]
```diff
@@ -643,15 +665,23 @@ private void ExecuteApplication(string path, string args, out string stdOut, out
...
-                Console.WriteLine($"IntegrationTestBase.Execute: Stopped {executableName}. Exit code = {0}", exitCode);
+                WriteLine("IntegrationTestBase.Execute: stdError = {0}", stdError);
+                WriteLine("IntegrationTestBase.Execute: stdOut = {0}", stdOut);
+                WriteLine($"IntegrationTestBase.Execute: Stopped {executableName}. Exit code = {0}", exitCode);
+            }
+        }
+
+        private void WriteLine(string format, params object[] arg)
+        {
+            if (TestContext == null)
```
Comment (nohwnd) [inline test/Microsoft.TestPlatform.TestUtilities/IntegrationTestBase.cs:682]: If test context is not null, are we not going to report the output to console anymore?

Comment (Haplois) [reply test/Microsoft.TestPlatform.TestUtilities/IntegrationTestBase.cs:682]: If we have a context, I redirect all output to a file and attach it to the test result itself. (or I was hoping to :))

**Comment by Haplois** [comment]
This PR is abandoned for now. When we switch to the Arcade, it will hopefully attach these outputs to our test result. The changes here will be needed at that time, but currently, they won't be attached.

**Pull request closed without merging**